src/pages/dar_application/DataAccessRequestApplication.jsx

```diff
@@ -568,7 +568,13 @@ const DataAccessRequestApplication = (props) => {
                   <ConditionalAccordion
                       condition={false}
                       title={`DAR Report ${reverseOrderedDARs.length}`}>
-                    <ProgressReportApplication readOnlyMode={false} datasets={filterForProgressReport(datasets, reverseOrderedDARs[0].datasetIds)} dar={reverseOrderedDARs[0]}/>
+                    <ProgressReportApplication
+                      readOnlyMode={false}
+                      datasets={filterForProgressReport(datasets, reverseOrderedDARs[0].datasetIds)}
+                      dar={reverseOrderedDARs[0]}
+                      location={props.location}
```

Comment:
Can we pull `location` out as a const instead? It makes it more difficult to identify where certain variables are used otherwise.